local/test-binary-parallel-mtr

```diff
@@ -333,6 +333,69 @@ if [[ "${CI_FS_MTR}" = 'yes' ]]; then
     echo KH,SUITE_TOTAL,CI_FS_tests,time,$runtime
 fi
 
+#
+# PS PROTOCOL tests
+if [[ "${WITH_PS_PROTOCOL}" = 'yes' ]]; then
+    echo "Running PS PROTOCOL tests"
+    start=`date +%s`
+
+    MTR_ARGS_PS=${MTR_ARGS}
+
+    if [[ $UNIT_TESTS == "1" ]]; then
+        MTR_ARGS_PS+=" --unit-tests-report"
+        UNIT_TESTS=0
+    fi
```

Comment:
This should be removed after https://github.com/Percona-Lab/ps-build/pull/401 was merged